doc/articles/interop/apple-login.md

```diff
@@ -0,0 +1,158 @@
+---
+uid: Uno.Tutorials.SignInWithApple
+---
+
+# Apple Sign-In Button with MAUI Embedding in Uno
+
+In Uno Platform projects, implementing a "Sign in with Apple" Button is possible through MAUI embedding (more on which in our [Overview](xref:Uno.Extensions.Maui.Overview)). Below is a simple example of how you can add the Apple Sign-In Button to your project. We will inject it into the visual tree from the code-behind and handle the authorization using Apple's Authentication Services.
+
+## Steps to Add "Sign in with Apple" Button
+
+Below is a simple example of how you can add the Apple Sign-In Button to your project. We will inject it into the visual tree from the code-behind and handle the authorization using Apple's Authentication Services.
```

Comment:
This text seems to be repeated. It looks very similar to the paragraph above.

Reply:
Good point, thanks! I removed the paragraph and restuctured the page a little bit.